packages/neuron-wallet/src/services/transactions.ts

```diff
@@ -290,7 +290,7 @@ export default class TransactionsService {
     const previousOutputs: OutputEntity[] = []
     for (const i of transaction.inputs!) {
       const input = new InputEntity()
-      input.outPointHash = i.previousOutput.hash
+      input.outPointTxHash = i.previousOutput.txHash
```

Comment:
It's this version of SDK is still `outPoint.txHash` or `outPoint.cell.txHash` ?

Reply:
> It's this version of SDK is still `outPoint.txHash` or `outPoint.cell.txHash` ?

prerelease of v0.11, which use outpoint of `{ txHash, index }`

Reply:
ok, i c